docs/resources/mlflow_webhook.md

```diff
@@ -0,0 +1,116 @@
+---
+subcategory: "MLflow"
+---
+# databricks_mlflow_webhook Resource
+
+This resource allows you to create [MLflow Model Registry Webhooks](https://docs.databricks.com/applications/mlflow/model-registry-webhooks.html) in Databricks.  Webhooks enable you to listen for Model Registry events so your integrations can automatically trigger actions. You can use webhooks to automate and integrate your machine learning pipeline with existing CI/CD tools and workflows. Webhooks allow trigger execution of a Databricks job or call a web service on specific event(s) that is generated in the MLflow Registry - stage transitioning, creation of registered model, creation of transition request, etc.
+
+## Example Usage
+
+### Triggering Databricks job
+
+```hcl
+variable "dbhost" {
+  description = "URL of Databricks workspace"
+}
+variable "dbtoken" {
+  description = "Token to access Databricks workspace"
+}
+
+data "databricks_current_user" "me" {}
+data "databricks_spark_version" "latest" {}
+data "databricks_node_type" "smallest" {
+  local_disk = true
+}
+
+resource "databricks_notebook" "this" {
+  path     = "${data.databricks_current_user.me.home}/MLFlowWebhook"
+  language = "PYTHON"
+  content_base64 = base64encode(<<-EOT
+    import json
+ 
+    event_message = dbutils.widgets.get("event_message")
+    event_message_dict = json.loads(event_message)
+    print(f"event data={event_message_dict}")
+    EOT
+  )
+}
+
+resource "databricks_job" "this" {
+  name = "Terraform MLflowWebhook Demo (${data.databricks_current_user.me.alphanumeric})"
+
+  new_cluster {
+    num_workers   = 1
+    spark_version = data.databricks_spark_version.latest.id
+    node_type_id  = data.databricks_node_type.smallest.id
+  }
+
+  notebook_task {
+    notebook_path = databricks_notebook.this.path
+  }
+}
+
+resource databricks_mlflow_webhook "job" {
```

Comment:
add quotes around resource type. try running `make fmt-docs` - maybe it'll do that for you automatically.

Reply:
done, also additional files were reformatted

Reply:
hm, it did not add quotes around resource type. definition must be `resource "databricks_mlflow_webhook" "job" {` ...

Reply:
I need to write 1000 times "don't multitask" :-)